src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,69 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  visiblePeople: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = ({ visiblePeople }) => {
+  const { slug } = useParams<{ slug: string }>();
```

Comment:
```suggestion
  const { slug } = useParams();
```
there is no need to add generics for useParams